### PR TITLE
added a new api for uploading the files into the local project directory

### PR DESCRIPTION
**Pull request by mshankarrao**

### CODE REVIEW

**Comment by samelamin** [comment]
Lets get this integrated on a page before merging please

Either on the edit brief page or the new brief form

But I want the files to be downloadable once someone explores the brief

File: api/src/frontend/tests/freelancers.test.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ function setUp() {
         "display_name": "test",
         "password": "test",
         "web3Accounts": [],
+        "web3_address": "test",
```
Comment (cuteolaf) [inline api/src/frontend/tests/freelancers.test.tsx:12]: "web3_address" is duplicated.

File: api/src/frontend/tests/freelancers.test.tsx [hunk 2]
```diff
@@ -22,6 +23,7 @@ test("test Freelancer rendering", () => {
...
```
Comment (cuteolaf) [inline api/src/frontend/tests/freelancers.test.tsx:25]: "web3_address" is duplicated.

**Review by cuteolaf — CHANGES_REQUESTED** [review]

File: api/src/frontend/tests/freelancers.test.tsx (hunk 1 above)
Comment (mshankarrao) [reply api/src/frontend/tests/freelancers.test.tsx:12]: Hey isn’t the user is configured that way in the models file
 export type User = {
    id: number;
    display_name: string;
    web3Accounts: Web3Account[];
    username: string;
    password?: string;
    getstream_token: string;
    web3_address: string;
};

Comment (cuteolaf) [reply api/src/frontend/tests/freelancers.test.tsx:12]: yeah,but in test code, it has two "web3_address" member in one object.
That's why I mentioned about it.

File: api/src/frontend/tests/freelancers.test.tsx (hunk 2 above)
Comment (mshankarrao) [reply api/src/frontend/tests/freelancers.test.tsx:25]: Ah got it thanks for pointing out

File: api/src/frontend/tests/freelancers.test.tsx (hunk 1 above)
Comment (mshankarrao) [reply api/src/frontend/tests/freelancers.test.tsx:12]: Got it and fixed it

**Pull request merged**